cpp/Platform.Interfaces.Tests/Platform.Interfaces.Tests.cpp

```diff
@@ -3,162 +3,162 @@
 
 namespace Platform::Interfaces::Tests
 {
-    TEST(CompileTests, ICounter)
+    TEST(CompileTests, Counter)
     {
-        struct EmptyCounter
+        struct EmptyCounter: ICounter<int>
```

Comment:
You can name this struct as `EmptyCounter1`

Reply:
Why `1`?

Reply:
Because there is `EmptyCounter2` next to it.

Reply:
Oh really, then move there to diff scope

Reply:
This can work too. Both solutions are fine.